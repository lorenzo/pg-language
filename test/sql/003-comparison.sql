SELECT 'ts' :: spoken_spoken_lang <  'tt' :: spoken_spoken_lang;

SELECT 'ts' :: spoken_spoken_lang <= 'tt' :: spoken_spoken_lang;

SELECT 'ts' :: spoken_spoken_lang >  'tt' :: spoken_spoken_lang;

SELECT 'tt' :: spoken_spoken_lang >  'ts' :: spoken_spoken_lang;

SELECT 'tt' :: spoken_spoken_lang >= 'ts' :: spoken_spoken_lang;

SELECT 'tt' :: spoken_spoken_lang >  'tt' :: spoken_spoken_lang;

SELECT 'tt' :: spoken_spoken_lang >= 'tt' :: spoken_spoken_lang;

SELECT 'tt' :: spoken_spoken_lang =  'ts' :: spoken_spoken_lang;

SELECT 'tt' :: spoken_spoken_lang != 'ts' :: spoken_spoken_lang;

SELECT 'tt' :: spoken_spoken_lang != 'tt' :: spoken_spoken_lang;

SELECT 'tt' :: spoken_spoken_lang =  'tt' :: spoken_spoken_lang;
