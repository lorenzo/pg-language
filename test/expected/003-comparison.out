SELECT 'ts' :: spoken_lang <  'tt' :: spoken_lang;
 ?column? 
----------
 t
(1 row)

SELECT 'ts' :: spoken_lang <= 'tt' :: spoken_lang;
 ?column? 
----------
 t
(1 row)

SELECT 'ts' :: spoken_lang >  'tt' :: spoken_lang;
 ?column? 
----------
 f
(1 row)

SELECT 'tt' :: spoken_lang >  'ts' :: spoken_lang;
 ?column? 
----------
 t
(1 row)

SELECT 'tt' :: spoken_lang >= 'ts' :: spoken_lang;
 ?column? 
----------
 t
(1 row)

SELECT 'tt' :: spoken_lang >  'tt' :: spoken_lang;
 ?column? 
----------
 f
(1 row)

SELECT 'tt' :: spoken_lang >= 'tt' :: spoken_lang;
 ?column? 
----------
 t
(1 row)

SELECT 'tt' :: spoken_lang =  'ts' :: spoken_lang;
 ?column? 
----------
 f
(1 row)

SELECT 'tt' :: spoken_lang != 'ts' :: spoken_lang;
 ?column? 
----------
 t
(1 row)

SELECT 'tt' :: spoken_lang != 'tt' :: spoken_lang;
 ?column? 
----------
 f
(1 row)

SELECT 'tt' :: spoken_lang =  'tt' :: spoken_lang;
 ?column? 
----------
 t
(1 row)

